api/api.yaml

```diff
@@ -124,6 +124,11 @@ definitions:
       search_placeholder_text:
         type: string
         description: Optional. What to show in the search box by default
+      time_series_unit:
+        type: string
+        description: >
+          Optional. The unit of time (such as month, year, etc.) with
```

Comment:
130-131:

If time_series_column in bigquery.json is set, this must be set. This is used to label the time axis -- eg Month or Year.

FYI search_placeholder_text is optional because if it's not set here, the UI will show something by default